### ISSUE TITLE
Org NuGet token is invalid for GraphQL.Relay package

### ISSUE THREAD

**Issue by Shane32** [graphql-dotnet/relay#110]
The organization nuget token, stored in the NUGET_AUTH_TOKEN organization secret, cannot be used to publish GraphQL.Relay to NuGet.  Either the auth token needs to be replaced/fixed (@joemcbride ) or another secret needs to be added to this project settings with the proper permissions (@jquense ?) before a new version of this project can be published to NuGet.

**Comment by joemcbride** [comment]
Correct, looks like @jquense is the current owner of the `GraphQL.Relay` package.

**Comment by jquense** [comment]
o boy it's been a long time since i've tried to get into nuget what do i need to do? is there a username of either of ya'll i can add

**Comment by Shane32** [comment]
@jquense yes my NuGet username is Shane32

**Comment by jquense** [comment]
added you to the gql.relay org

**Comment by Shane32** [comment]
Fixed.  Released 0.6.2 on nuget.

**Comment by sungam3r** [comment]
https://github.com/graphql-dotnet/relay/runs/7076751656?check_suite_focus=true#step:10:13  again

**Comment by Shane32** [comment]
I can fix it

**Comment by sungam3r** [comment]
Thanks.
